docs/command_line_reference.rst

```diff
@@ -663,7 +663,7 @@ By default, the command line reporter will print the results only on standard ou
 
 With this option you can customize Rally's internal Elasticsearch client.
 
-It accepts a list of comma-separated key-value pairs. The key-value pairs have to be delimited by a colon. These options are passed directly to the Elasticsearch Python client API. See `their documentation on a list of supported options <http://elasticsearch-py.readthedocs.io/en/master/api.html#elasticsearch.Elasticsearch>`_.
+It accepts a list of comma-separated key-value pairs. The key-value pairs have to be delimited by a colon. These options are passed directly to the Elasticsearch Python client API. See `their documentation on a list of supported options <https://elasticsearch-py.readthedocs.io/en/v7.14.0/api.html#elasticsearch>`_.
```

Comment:
For the sake of not having to remember changing the version here when we upgrade `elasticsearch-py` could we leverage a variable, similar to how we do for [MIN_PY_VER](https://github.com/elastic/rally/blame/master/docs/install.rst#L30)? This is achieved with real python code on [docs/conf.py](https://github.com/elastic/rally/blob/a921dccd3158f3821d36d74bcc9e19b629594f1e/docs/conf.py#L49-L59).

Reply:
Perfect, didn't even occur to me. I'll do that.

Reply:
Done in https://github.com/elastic/rally/pull/1554/commits/8fb9a3dbe2e9e1b4cf369deb43a857307495e46a